source/stylesheets/base/_variables.scss

```diff
@@ -1,15 +1,18 @@
 // Typography
 $base-font-family: $font-stack-system;
-$heading-font-family: $base-font-family;
+$heading-font-family: "Ahkio-regular";
+$base-font-size: 16px;
 
 // Line height
 $base-line-height: 1.5;
 $heading-line-height: 1.2;
 
 // Other Sizes
 $base-border-radius: 3px;
+$large-border-radius: 30px;
 $base-spacing: 50px;
```

Comment:
This is a really large base-spacing. Typically, I'd set it as like 150% of the base-font size. Does it need to be this big?